editor/modes/visual-editor/index.js

```diff
@@ -10,17 +11,23 @@ import './style.scss';
 import Inserter from 'components/inserter';
 import VisualEditorBlock from './block';
 
-function VisualEditor( { blocks } ) {
+function VisualEditor( { blocks, blockOrder } ) {
```

Comment:
>`blockOrder` is not being use in this component, shoud we move it the the `connect` of the `VisualEditorBlock`   component?

Yep! And in fact, the `order` prop is already specified in `block.js`'s `connect`, so just need to remove this.